### DISCUSSION TITLE
Migrating Package/@Comments

### DISCUSSION THREAD

**Discussion by djohnst10** [wixtoolset/issues#7419] [Questions] [answered]
I've done the "Upgrade to WiX v4..." approach of converting our WiX v3 project but the existing Comments attribute of the Package element did not get migrated. The Comments attribute used to populate the Summary Information Comments field, now it is populated with the following "This installer database contains the logic and data required to install [ProductName]."

Looking at the documentation, I don't see where the Comments attribute went. Why was it removed?

**Comment by barnson** [comment, marked as answer]
https://github.com/wixtoolset/issues/issues/7369

**Comment by robmen** [comment]
What did you set it to before?

**Reply by djohnst10** [reply]
It was a string like this:
!(wix.DebugOrRelease) !(wix.ProductVersion). Built !(wix.CurrentDateTime).

CurrentDateTime was generated in the project file like so:

```
<Target Name="BeforeBuild">
  <PropertyGroup>
    <WixVariables>$(WixVariables);CurrentDateTime=$([System.DateTime]::Now);</WixVariables>
  </PropertyGroup>
</Target>
```

**Reply by jcasale** [reply]
I also wrote useful information in the comments that I did not want in the name, such as the version.
